src/components/Disputes/Import.js

```diff
@@ -68,45 +70,62 @@ function Import() {
   async function handleProjectCreationn() {
     let nameInput = inputName.current.value
     let budgetInput = inputBudget.current.value
+    let dropdownInput = inputDropdown.current.value
     console.log(nameInput, budgetInput)
-    await create('123', nameInput, budgetInput)
+    await create(dropdownInput, nameInput, budgetInput)
   }
 
   return (
     <>
-      {wallet.account !== null ? (
+      {wallet.account !== null &&
+      localStorage.getItem('ACCESS TOKEN') !== null ? (
```

Comment:
Make the button disabled (not clickable) instead of returning null. Because the user has to know why it is not clickable.

Reply:
AragonUI doesn't provide disabled button prop, one strategy would be to create a pop up and a separate function when clicked

Reply:
Alright! You can then move this condition to the side panel. Display an error message in the side panel similar to the profile page asking to connect to the GitHub account.

Reply:
Fixed